elasticsearch/repo.sls

```diff
@@ -1,10 +1,11 @@
 # Install Elasticsearch repository
 {%- set repo_version = salt['pillar.get']('elasticsearch:repo_version', '1.1') %}
+{%- set repo_url = salt['pillar.get']('elasticsearch:repo_url', "deb http://packages.elastic.co/elasticsearch/{{ repo_version }}/debian stable main") %}
 
 elasticsearch_repo:
   pkgrepo.managed:
     - humanname: Elasticsearch Repo
-    - name: deb http://packages.elastic.co/elasticsearch/{{ repo_version }}/debian stable main
+    - name: {{ repo_url }}
     - file: /etc/apt/sources.list.d/elasticsearch.list
     - keyid: D88E42B4
```

Comment:
Elasticsearch still signs their packages with the same key, so not really necessary. 🌴  https://www.elastic.co/guide/en/elasticsearch/reference/5.2/deb.html